distribution/docs/src/main/resources/_contents/_tables/ddf.catalog.registry.policy.RegistryPolicyPlugin-table-contents.adoc

```diff
@@ -0,0 +1,55 @@
+.[[ddf.catalog.registry.policy.RegistryPolicyPlugin]]Registry Policy Plugin
```

Comment:
Just realized that this file should be replaced with the `org.codice.ddf.catalog.registry.policy.RegistryPolicyPlugin-table-contents.adoc`